dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -1490,4 +1336,165 @@ Set getContainers() {
 	Set getFolders() {
 		return folders;
 	}
+
+	private class DependencyProcessorItem {
+		String assetKey;
+		AssetTypes assetTypes;
+
+		public DependencyProcessorItem(String assetKey, AssetTypes assetTypes) {
+			this.assetKey = assetKey;
+			this.assetTypes = assetTypes;
+		}
+	}
+
+	private class DependencyProcessor {
+		private BlockingQueue<DependencyProcessorItem> queue;
+		private Map<AssetTypes, Set<String>> sets;
+		private boolean finish;
+		private boolean started;
+		private List<DependencyThread> threads;
+
+		void put(final String assetKey, final AssetTypes assetTypes) {
```

Comment:
Rename to `assetType`. The enum should have been named singular, `AssetType` so let's at least name the variable singular.

Reply:
done https://github.com/dotCMS/core/pull/19955/commits/867f260028f48d4ab70b7b3d692666eae8ee3f0e#diff-dcfa32b94c8ab208391686a086d1fa0f4ba566eeff3b1f81c3787e0efa300e8bR10005